pkg/plugins/backendplugin/provider/provider.go

```diff
@@ -10,26 +10,29 @@ import (
 	"github.com/grafana/grafana/pkg/plugins/backendplugin/pluginextensionv2"
 	"github.com/grafana/grafana/pkg/plugins/backendplugin/secretsmanagerplugin"
 	"github.com/grafana/grafana/pkg/plugins/log"
+	"github.com/grafana/grafana/pkg/services/featuremgmt"
```

Comment:
We can do that, yes, but we still need to import that package to get the feature flags:

```go
features.IsEnabled(featuremgmt.FlagPluginsSkipHostEnvVars)
```

`FlagPluginsSkipHostEnvVars` (which is auto-generated) is only declared in `github.com/grafana/grafana/pkg/services/featuremgmt`.

I will change so it uses the `plugins.FeatureToggles` interface but unfortunately it won't remove the import

Reply:
Is a pity, but we have options if we want to work around that eventually. Is fine for now then 👍

Reply:
I went a bit down on a rabbit hole [on a separate branch](https://github.com/grafana/grafana/commit/82c438b3088ac2a795f47f71001445820754d36b) trying to put feature flags into the plugins package. It requires some changes that aren't strictly related to this PR (the [plugin-specific interface](https://github.com/grafana/grafana/blob/e855efb13de80615d14bdcbf11b64a8ab5b74260/pkg/plugins/ifaces.go#L157-L160) we is different compared to [featuremgmt.FeatureToggles](https://github.com/grafana/grafana/blob/ade140c16141ea7d5dbdef753cfdd9365c172257/pkg/services/featuremgmt/models.go#L8-L10), and wire is complaining).

I think I will leave this refactoring work for another PR because it will require some changes in other places as well. I think it's easier if we keep using `featuremgmt.FeatureToggles` in this PR, after all we still have to import `featuremgmt` for the constant definition.

WDYT?

Reply:
Yeah that's cool 👍